packages/legacy/core/App/screens/PINExplainer.tsx

```diff
@@ -50,7 +50,7 @@ const PINExplainer: React.FC<PINExplainerProps> = ({ continueCreatePIN }) => {
             <Trans
               i18nKey="PINCreate.Explainer.PINReminder"
               components={{
-                b: <Text style={TextTheme.labelTitle} />,
+                b: <Text style={TextTheme.bold} />,
```

Comment:
This is to use the correct font size so that the bolded text within the `b` isn't slightly smaller than the rest of the sentence.